native/src/routes/Pois.tsx

```diff
@@ -157,9 +157,9 @@ const Pois = ({
   const featureLocations = pois
     .map(poi => poi.featureLocation)
     .filter((feature): feature is Feature<Point> => !!feature)
-  const locId = (route?.params as RouteParams)?.locationId
+
   const currentFeature: Feature<Point> | undefined = featureLocations.find(
-    feature => feature.properties?.id === Number(locId)
+    feature => feature.properties?.id === Number(route.params.selectedPoiId)
   )
```

Comment:
If you want you can remove the type annotation

Reply:
no the the ```properties``` are typed as following by 'geojson'
```[name:string]:any```
In this case ```id : number```
We could override the interface to get proper type checking